packages/bedrock-components/src/components/sqb-conditional-section/sqb-conditional-section.feature

```diff
@@ -3,41 +3,51 @@
 
 Feature: Conditional Section
 
-  Conditional Section adds JSONata conditional logic that can display or hide children components based on user segments or custom fields
+  Conditional Section adds JSONata conditional logic that can display or hide children components based on user segments, custom fields or email
 
   @motivating
-  Scenario: Content is hidden or shown depending on user segments
+  Scenario: Content can be hidden or shown depending on user segments
     Given the component's condition prop is set to "'vip' in user.segments"
     And the component contains the text "Hello"
     And the user has no segments
     When the widget is loaded
-    Then "Hello" will not be shown
+    Then "Hello" is not shown
     When the user is added to segment "vip"
-    Then "Hello" will be shown
+    Then "Hello" is shown
 
   @motivating
-  Scenario: Content is hidden or shown depending on user's custom fields
+  Scenario: Content can be hidden or shown depending on user's custom fields
     Given the component's condition prop is set to "user.customFields.foo = true"
     And the component contains the text "Hello"
     And the user's custom field "foo" is false
     When the widget is loaded
-    Then "Hello" will not be shown
+    Then "Hello" is not shown
     When the user's custom field "foo" is set to true
-    Then "Hello" will be shown
+    Then "Hello" is shown
+
+  @motivating
+  Scenario: Content can be hidden or shown depending on user's email
+    Given the component's condition prop is set to "$not('example.com' in $substringAfter(user.email, '@'))"
```

Comment:
Would have been convenient to include `emailDomain` as a thing separate from `email` if we're getting customers to build logic on it.

```suggestion
    Given the component's condition prop is set to "$not('example.com' = user.emailDomain)"
```